Automattic/jetpack
# Plans: update plan constants and icons

Fixes p9F6qB-c2B-p2

## Proposed changes:

* Adds WoA 3 year plan slugs, and defines correct plan icons
Currently if a WoA 3 year plan, such as the Business or eCommerce plan is active, the plan icon for the `ecommerce-bundle-3y` and `business-bundle-3y` is not defined, and results in infinite loading of the Jetpack > Plans page in WP-Admin on WoA site

![image](https://github.com/Automattic/jetpack/assets/65082164/fed3e1a1-a81f-4bb5-8ea3-e246ecb70425)


![image](https://github.com/Automattic/jetpack/assets/65082164/84b9b25f-de31-4fd5-b706-96c0963b8171)


### Other information:


## Jetpack product discussion


p9F6qB-c2B-p2

## Does this pull request change what data or activity we track or use?
No.

## Testing instructions:






* Install the Jetpack Beta Tester plugin
* Use Jetpack Beta Tester plugin to find and activate this branch `add/WoA-3-year-plan-icons`
* Add a 3 year plan such as the Business or eCommerce plan. Deactivate Jetpack, reactivate Jetpack (reconnect if necessary), and ensure the correct plan slug is showing in WP CLI `ecommerce-bundle-3y` or `business-bundle-3y`
* Navigate to Jetpack > Plans in WP-Admin, and ensure that the plans page loads
